x-pack/plugins/infra/public/components/asset_details/tabs/overview/section/collapsible_section.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiAccordion, EuiFlexGroup, EuiFlexItem, useGeneratedHtmlId } from '@elastic/eui';
+import React, { useState } from 'react';
+
+export type SectionTriggerValue = 'closed' | 'open';
+
+export const CollapsibleSection = ({
+  title,
+  closedSectionContent,
+  extraAction,
+  children,
+  shouldCollapse,
+  ['data-test-subj']: dataTestSubj,
+  id,
+}: {
+  title: React.FunctionComponent;
+  closedSectionContent?: React.ReactNode;
+  extraAction?: React.ReactNode;
+  dependsOn?: string[];
+  children: React.ReactNode;
+  shouldCollapse: boolean;
+  ['data-test-subj']: string;
+  id: string;
+}) => {
+  const [trigger, setTrigger] = useState<SectionTriggerValue>('open');
+
+  const Title = title;
+  const ButtonContent = () =>
+    closedSectionContent && trigger === 'closed' ? (
+      <EuiFlexGroup gutterSize="m">
+        <EuiFlexItem grow={false}>
+          <>
+            <Title />
+          </>
```

Comment:
Is this extra Fragment necessary?
```suggestion
            <Title />
```